src/store/project.ts

```diff
@@ -281,11 +303,32 @@ export const projectStore = createPartialStore<ProjectStoreTypes>({
             }
           }
 
+          if (
+            semver.satisfies(
+              projectAppVersion,
+              "<0.16.2",
+              semverSatisfiesOptions
+            )
+          ) {
+            // 0.16.2 未満のプロジェクトファイルはトークモードの情報(audioKeys/audioItems)しか
+            // 保存されていなかったので、それをtalkに移動する
```

Comment:
細かいけどここも指示語入ってますね。言いたいことはこう？
```suggestion
            // 0.16.2 未満のプロジェクトファイルはトークのみ
            // なので全情報(audioKeys/audioItems)をtalkに移動する
```